### ISSUE TITLE
Is it possible to writestream into multiple sheets in a given file?

### ISSUE THREAD

**Issue by sdawson26** [STRML/node-xlsx-writestream#15]
In the README, you mention there is an empty "options" object placed here:

XLSXWriter('spreadsheet.xlsx', {} /*options*/);

What kind of options are available? Can I use that to write data into a multiple sheets within the file?

**Comment by AMRITPK** [comment]
did you get any solution for this? facing same

**Comment by ashokduraik** [comment]
I too need a solution for this. Please do reply

**Comment by yamila-fraiman** [comment]
Any solution?

**Comment by baranovxyz** [comment]
No, it is not possible to write to several sheets simultaneously. But it is possible to fully write the first sheet, then the second sheet, then third, then finish the book. It will not be possible to again write to first sheet.

**Comment by cscan** [comment]
Add me in
